apps/client/src/common/components/view-params-editor/InlineColourPicker.tsx

```diff
@@ -20,13 +20,9 @@ export default function InlineColourPicker(props: InlineColourPickerProps) {
   const { name, value } = props;
   const [colour, setColour] = useState(() => ensureHex(value));
 
-  const debouncedChange = (value: string) => {
-    setColour(value);
-  };
-
   return (
     <div className={style.inline}>
-      <PopoverPicker color={colour} onChange={debouncedChange} />
+      <SwatchPicker color={colour} onChange={setColour} />
```

Comment:
question: I wonder if here we should actually show the value of the colour. 

Users will often want the HEX value to key externally
This will mean having  a span element with the colour
It would also make more sense to the inline styles and naming

something like (untested):


```suggestion
      <SwatchPicker color={colour} onChange={setColour} />
      <span>{colour}</span>
```

Reply:
For this, I believe the first option seems good. We already have an input from the `<SwatchPicker>` so that could work. This can be specifically to view the current value externally